src/ui/RealmBrowser/AddClassModal/AddClassModal.tsx

```diff
@@ -136,6 +154,29 @@ export const AddClassModal = ({
               </InputGroupAddon>
             </InputGroup>
           </FormGroup>
+          {primaryKey && (
+            <Card>
+              {/* Todo: Wording */}
+              <CardBody>
+                <CardSubtitle>MongoDB Sync:</CardSubtitle>
+                <CardText>
+                  <small>
+                    {
+                      'If this Realm is intended to be synced with MongoDB, ensure to keep the primary key name "_id", with the type "ObjectId".'
```

Comment:
In principle I like that we're using the `ObjectId` here, since thats the name of the BSON type - but .. at the same time we're using `object id` a couple of other places ... it feels inconsistent, but perhaps the solution is to lobby that this changes to `ObjectId` in Realm JS?

Reply:
It's coming, but might end up being `objectId `, we need to streamline either way.
I'll stick with 'object id', that's pretty ease to find.